akita_inu_asa_utils.py

```diff
@@ -51,6 +55,7 @@ def compile_program(client, source_code, file_path=None):
     else:
         check_build_dir()
         dump(base64.b64decode(compile_response['result']), 'build/' + file_path)
+    return compile_response["hash"]

```

Comment:
I recommend making a separate function for returning the hash of compile_response so you could get the hash without dumping the result.

Reply:
Is the hash actually used or was it in place just for debugging?

Reply:
I was using this initially, but in the end I took the address directly from the LogicSigAccount object instead. It's still [used as a sanity check](https://github.com/soimec/SmartContracts/blob/8db7a519b38cbf196d3f3d3c8f2bfd8a239c832c/test/nft_royalties_test.py#L56-L57) in testing, but otherwise can be removed or done in a separate function. Let me know what you think is preferable.

Reply:
Correction, actually, the hash value is still used in deployment. I'll put it into its own function.